src/main/java/seedu/address/model/Model.java

```diff
@@ -117,6 +118,11 @@ public interface Model {
      */
     BigFraction getBalance(Name name);
 
+    /**
+     * Gets all names in the model.
+     */
+    Set<Name> nameSet();
```

Comment:
Consider changing the method name to start with a verb such as `getNameSet` or `getAllNames`